db/version_edit.h

```diff
@@ -278,6 +281,11 @@ class VersionEdit {
     return new_files_;
   }
 
+  void MarkGroupCommit(uint32_t remaining_entries) {
```

Comment:
To me "group commit" means any group of `VersionEdit`s that were `fsync`'d together when they were written, which doesn't necessarily mean they need to be applied atomically during recovery. I think `MarkGroupCommit` needs to be invoked only for atomic CF flush, otherwise we would have a forward compatibility issue. Maybe call it "MarkFragmentsRemaining" or something to indicate these `VersionEdit`s cannot be applied in a standalone way.

Reply:
SG, will update the PR.